warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandCompositeTerms.java

```diff
@@ -39,7 +44,7 @@
 import org.apache.commons.jexl2.parser.ASTReference;
 import org.apache.commons.jexl2.parser.ASTReferenceExpression;
 import org.apache.commons.jexl2.parser.JexlNode;
-import org.apache.log4j.Logger;
```

Comment:
@brianloss  I don't know datawave's desire to move to slf4j....

Reply:
Nor do I. I prefer it, but I'm not aware of people's plans. I definitely used it (and log4j2) in the microservices.